scripts/zones/Castle_Oztroja/Zone.lua

```diff
@@ -37,24 +40,12 @@ end;
 function onRegionEnter(player,region)
 end;
 
-Oz_Handle_Table =
-{
-    [0] = {[17396196] = 8, [17396197] = 8, [17396198] = 8, [17396199] = 8},
-    [1] = {[17396196] = 8, [17396197] = 9, [17396198] = 9, [17396199] = 9},
-    [2] = {[17396196] = 9, [17396197] = 8, [17396198] = 9, [17396199] = 9},
-    [3] = {[17396196] = 9, [17396197] = 8, [17396198] = 8, [17396199] = 9},
-    [4] = {[17396196] = 9, [17396197] = 8, [17396198] = 9, [17396199] = 8},
-    [5] = {[17396196] = 8, [17396197] = 8, [17396198] = 9, [17396199] = 8},
-    [6] = {[17396196] = 9, [17396197] = 9, [17396198] = 8, [17396199] = 9},
-    [7] = {[17396196] = 9, [17396197] = 9, [17396198] = 9, [17396199] = 8},
-    [8] = {[17396196] = 8, [17396197] = 8, [17396198] = 9, [17396199] = 9},
-};
-
 function onGameHour(zone)
     local VanadielHour = VanadielHour();
-    if (VanadielHour % 24 == 0) then -- Change handles every game day
+    if (VanadielHour % 24 == 0) then -- Change handles and passwords every game day
         Oz_handleSet = math.random(0,8);
```

Comment:
im not sure we want `Oz_handleSet` hanging around as global variable as we dont have a use for it outside this if block
using a `local` variable may be more appropriate as it wont stay around in memory longer than we need it to

Reply:
Needs to be global so other files can use it

Reply:
unless you can make it work a better more efficient way... this is pretty much all i have

Reply:
ahh sorry i didnt see it being used in the other scripts
yeah i cant think of a better way until we get round to redoing quests like in quest_rewrite branch, no need to change this